apps/src/templates/SchoolDataInputs.jsx

```diff
@@ -29,8 +29,13 @@ export default function SchoolDataInputs({
   const [country, setCountry] = useState('');
   const [zipSearchReady, setZipSearchReady] = useState(false);
 
-  let COUNTRY_ITEMS = [{value: 'selectCountry', text: i18n.selectCountry()}];
-  for (const item of Object.values(COUNTRIES)) {
+  let COUNTRY_ITEMS = [
+    {value: 'selectCountry', text: i18n.selectCountry()},
+    {value: 'US', text: 'United States'},
```

Comment:
Should 'United States' be a translatable string?

Reply:
I'm happy to add one- curious about the status of the current discussion with Jorge about this. For the rest of the countries I'm using the strings from the geographyConstants.js file (which are not translated). Do we have a plan to move toward a solution where all the countries are in their own locale? Otherwise, the US will be the only translated option.

Reply:
hm interesting! Yeah, that might be a question for Jorge and possibly a follow-up in a different PR

Reply:
Okay I found [where I was reading about it](https://codedotorg.atlassian.net/browse/ACQ-1720)! Not entirely the same but very relevant. I think for now I will add a string to translate 'United States' and add a comment to this ticket about other places on our site where all countries are displayed.